test/e2e/lua/dynamic_configuration.go

```diff
@@ -92,6 +92,60 @@ var _ = framework.IngressNginxDescribe("Dynamic Configuration", func() {
 			Expect(nginxConfig).Should(Equal(newNginxConfig))
 		})
 
+		It("handles endpoints only changes (down scaling of replicas)", func() {
```

Comment:
can you explain why is this necessary given you have https://github.com/kubernetes/ingress-nginx/pull/3628/files#diff-8700c46a8379c603b5e41ccbaff6b71eR124?

Reply:
Seems like this is an unnecessary remnant from a first version of the test. I removed it.